mono/mini/mini-llvm.c

```diff
@@ -8773,7 +8793,8 @@ static IntrinsicDesc intrinsics[] = {
 	{INTRINS_SSE_PAVGB, "llvm.x86.sse2.pavg.b"},
 	{INTRINS_SSE_PAUSE, "llvm.x86.sse2.pause"},
 	{INTRINS_SSE_DPPS, "llvm.x86.sse41.dpps"},
-	{INTRINS_SSE_ROUNDPD, "llvm.x86.sse41.round.pd"}
+	{INTRINS_SSE_ROUNDSS, "llvm.x86.sse41.round.ss"},
```

Comment:
I decided to use `round.ss` instead of `round.ps` because `round.ps` generated weird movs (it works nice for double but didn't work for float). CoreCLR also uses `round.ss` (scalar instead of packed).